blog/_posts/2016-09-20-julia-0.5-highlights.md

```diff
@@ -0,0 +1,726 @@
+---
+layout: post
+title:  Julia 0.5 Highlights
+author: Stefan Karpinski
+---
+
+Julia 0.5 is a pivotal release.
+It introduces more transformative features than any release since the first official version.
```

Comment:
I think those people will complain anyway. FWIW I thought that sentence made an exciting introduction to the post.